### ISSUE TITLE
Security: Update uglify dependency

### ISSUE THREAD

**Issue by smyth64** [marcuswestin/require#11]
Hey guys,
Today I run my ESLint and it told me hat there is the change of a Denial of Service attack.

![image](https://cloud.githubusercontent.com/assets/3246782/21233263/e04cb452-c2ee-11e6-9f26-341e1c9a1b64.png)

I think we should update uglify to a newer version.

**Comment by seanmcelroy** [comment]
Agreed.  Snyk.io is also alerting on this as well with the following text:

"[Tom MacWright](https://github.com/mishoo/UglifyJS2/issues/751) discovered that UglifyJS versions 2.4.23 and earlier are affected by a vulnerability which allows a specially crafted Javascript file to have altered functionality after minification. This bug was [demonstrated](https://zyan.scripts.mit.edu/blog/backdooring-js/) by [Yan](https://twitter.com/bcrypt) to allow potentially malicious code to be hidden within secure code, activated by minification."

Remediation
Upgrade UglifyJS to version 2.4.24 or higher.

It appears the latest version of UglifyJS is 3.3.14: https://www.npmjs.com/package/uglify-js

**Comment by outlawpractice** [comment]
Is this ever going to get released? Looks like it was fixed last year, but no new version published.
